src/component/app/_app.scss

```diff
@@ -1,5 +1,45 @@
 @import 'vars';
 
 .app {
-  padding: $pad * 2;
+  // margin-top: 5px;
+  padding: $medium-pad;
+}
+
+.nav-wrapper {
+  position: relative;
+  margin-left: auto;
+  text-align: right;
+  height: 40px;
+}
+
+.saved-link, .login-logout-button {
+  position: absolute;
+  margin: 0.5vw;
+}
+
+.login-logout-button {
+  top: 0;
+  right: 0;
+  width: 40%;
+  @media only screen and (min-width: 600px) {
+    right: 0;
```

Comment:
Doesn't look like this `right: 0` is needed, since it has that before the media query
@SethDonohue

Reply:
These must have been left over from me playing around in css yesterday eve. It took a bit to get the dang buttons where they are.